### PR TITLE
chore: fix typos and grammar

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: None
Ticket: None

### CODE REVIEW

File: 06.Artifact-creation/10.CI-CD/docs.md [hunk 1]
```diff
@@ -42,15 +42,15 @@ You can read about artifact creation details [here](../01.Create-an-Artifact/doc
...
 
 This scenario uses `mender-cli` tool to upload your Mender Artifact to the Mender server. 
 
-The input for the scenario is the Mender Artifact generated in the previous stage, and the output is upload result.
+The input for the scenario is the Mender Artifact generated in the previous stage, and the output is the upload result.
 
 You can read about uploading the Mender Artifact to the file storage [here](../../08.Server-integration/01.Using-the-apis/docs.md#set-up-mender-cli).
 
-You can use [Personal Access Tokens](../../08.Server-integration/01.Using-the-apis/docs.md#personal-access-tokens) with necessary permissions to perform automated actions on Mender server on your behalf.
+You can use [Personal Access Tokens](../../08.Server-integration/01.Using-the-apis/docs.md#personal-access-tokens) with necessary permissions to perform automated actions on the Mender server on your behalf.
```
Comment (merlin-northern) [inline 06.Artifact-creation/10.CI-CD/docs.md:53]: we rarely say "Mender server" usually ti is "Mender _S_erver"

**Review by merlin-northern — APPROVED** [review]
general comment: when we were rewriting the docs there was a rule: always there is a native English speaker reviewing. what happened to that?

File: 06.Artifact-creation/10.CI-CD/docs.md (hunk 1 above)
Comment (tranchitella) [reply 06.Artifact-creation/10.CI-CD/docs.md:53]: Will follow-up with a dedicated PR.

**Comment by tranchitella** [comment]
See: https://github.com/mendersoftware/mender-docs/pull/2063

**Pull request merged**